torch/onnx/symbolic_opset9.py

```diff
@@ -186,7 +186,7 @@ def cat(g, tensor_list, dim):
 
 @parse_args('v', 'i')
```

Comment:
Please also update the code in squeeze/unsqueeze symbolics to use the helper function.